### ISSUE TITLE
Model accuracy with Jester dataset is poor

### ISSUE THREAD

**Issue by joyalbin** [ahmetgunduz/Real-time-GesRec#42]
Hi I have tried to validate the pretrained model with Jester dataset.

Preconditions:
1. Retrained model used `jester_resnext_101_RGB_32.pth`
2. Dataset `Jester`
3. Configurations [opts.zip](https://github.com/ahmetgunduz/Real-time-GesRec/files/3841257/opts.zip)
4. Source modifications [diff.zip](https://github.com/ahmetgunduz/Real-time-GesRec/files/3841273/diff.zip)
5. PyTorch version `1.1.0`
6. Python version `3.7.3`

Test:
1. `python utils/jester_json.py 'annotation_Jester'` to prepare the dataset
2. `python offline_test.py` to start the execution

But the output precision is very poor

> [11/3721]	Time 1.07421 (1.13381)	prec@1 0.03409 prec@5 0.20455	precision 0.00000 (0.03213)	recall 0.00000 (0.01278)
[12/3721]	Time 1.09013 (1.13017)	prec@1 0.03646 prec@5 0.20312	precision 0.03030 (0.03198)	recall 0.03030 (0.01424)
[13/3721]	Time 1.07996 (1.12631)	prec@1 0.03365 prec@5 0.20192	precision 0.00000 (0.02952)	recall 0.00000 (0.01315)
[14/3721]	Time 1.08615 (1.12344)	prec@1 0.03125 prec@5 0.20089	precision 0.00000 (0.02741)	recall 0.00000 (0.01221)

Could you please help me to find what am missing to get the proper output?
Regards,
Albin

**Comment by ahmetgunduz** [comment]
You need to provide the model path as `resume_path`. Since you are providing as a `pretrain_path` the first convolution kernel is randomly generated regardless of the model size or modality of the inputs.
